app/lua/lobject.h

```diff
@@ -380,7 +380,7 @@ typedef union Closure {
 */
 
 #define CommonTable CommonHeader; \
-                    lu_byte flags; lu_byte lsizenode; struct Table *metatable;
+                    lu_byte flags; lu_byte lsizenode; struct Table *metatable
 
```

Comment:
Seems OK but unlikely to be Windows-specific?

Reply:
the Microsoft windows compiler seems to be more strict about extra semicolon here.

Reply:
No.  The original is incorrect as this will preprocess to have a double semi-colon inside a struct definition.  The `CommonTable` definition should not have a terminating semi-colon.

Reply:
Ah ha!  Thanks for clarification. :)